feedback/views.py

```diff
@@ -505,11 +505,18 @@ def get_context_data(self, **kwargs):
         update_context_for_feedbacks(self.request, context)
         return context
 
+    def get_paginate_by(self, queryset):
+        value = self.request.GET.get('paginate_by')
+        if value is not None and value.isnumeric() and (value == "50" or value == "100" or value == "200"):
```

Comment:
```suggestion
        if value is not None and value in ("50", "100", "200"):
```
This looks simpler. `isnumeric()` is unnecessary when the equality comparison includes only numeric strings.